components/organisms/Layout.js

```diff
@@ -20,22 +22,56 @@ export const Layout = ({
   locale,
   langUrl,
 }) => {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
   const { t } = useTranslation("common");
   const language = locale === "en" ? "fr" : "en";
+
   return (
     <div className="overflow-x-hidden">
       <header>
         <PhaseBanner phase={t("Alpha")}>{t("alphaText")}</PhaseBanner>
-        <div className="layout-container">
-          <Link key={language} href={langUrl} locale={language}>
-            <a data-cy="toggle-language-link">
-              {language === "en" ? "English" : "Français"}
-            </a>
-          </Link>
-          <Header
-            headerLogoAltText="Symbol of the Government of Canada"
-            headerLogoImage="/sig-blk-en.svg"
-          />
+        <div className="layout-container ">
+          <div className="flex-col flex lg:flex lg:flex-row justify-between lg:mt-2 ">
+            <div className="mt-4 lg:flex lg:flex-col flex flex-row justify-between ">
+              <Header
+                headerLogoAltText="Symbol of the Government of Canada"
+                headerLogoImage={
+                  language === "en" ? "/sig-blk-en.svg" : "/sig-blk-fr.svg"
+                }
+              />
+              <div className="visible lg:invisible ml-6 xs:ml-16">
```

Comment:
is this div nessecary ? Can you add these styles to the a element and achieve the same results ?

Reply:
Moved it to the img elements.